docs/installation.rst

```diff
@@ -53,43 +53,87 @@ The Superset web server and the Superset Celery workers (optional)
 are stateless, so you can scale out by running on as many servers
 as needed.
 
-Start with Docker
+Install and Deploy Superset Locally with Docker
 -----------------
 
-.. note ::
-    The Docker-related files and documentation are actively maintained and
-    managed by the core committers working on the project. Help and contributions
-    around Docker are welcomed!
+To try Superset locally, the
+best-supported currently method is via Docker, using ``docker-compose``. Superset
+does not have official support for Windows, so we have provided a VM
+workaround below. (We will update this documentation once Windows is
+supported.)
+
+**Step 0 - Install a Docker Engine and Docker Compose** 
+
+*Mac OSX:*
+        
+    `Install Docker for Mac <https://docs.docker.com/docker-for-mac/install/>`__, which includes the Docker engine and a recent version of `docker-compose` out of the box.
+    
+    Once you have Docker for Mac installed, open up the preferences pane for Docker, go to the "Resources" section and increase the allocated memory to 6GB. With only the 2GB of RAM allocated by default, Superset will fail to start.
+
+
+*Linux:*
+
+    `Install Docker on Linux <https://docs.docker.com/engine/install/>`__ by following Docker’s instructions for whichever flavor of Linux suits you.
+    
+    Because ``docker-compose`` is not installed as part of the base Docker installation on Linux, once you have a working engine, follow the `docker-compose installation instructions <https://docs.docker.com/compose/install/>`__ for Linux.
+
+
+*Windows:*
+
+    For Windows users, the best option may be to install an Ubuntu Desktop VM via `VirtualBox <https://www.virtualbox.org/>`__ and proceed with the Docker on Linux instructions inside of that VM. It is recommended to assign at least 8GB of RAM to the virtual machine as well as provisioning a hard drive of at least 40GB, so that there will be enough space for both the OS and all of the required dependencies.
```

Comment:
Let's add a note that Windows is not a supported execution environment by the community :)

Reply:
Good call. Noted added. PTAL